include/hse/types.h

```diff
@@ -74,6 +75,19 @@ enum hse_err_ctx {
  */
 struct hse_kvdb;
 
+#define HSE_MCLASS_CAPACITY_NAME "capacity" /**< Capacity media class name. */
+#define HSE_MCLASS_STAGING_NAME  "staging"  /**< Staging media class name. */
+
+/** @struct hse_mclass_info
+ * @brief Media class information.
+ */
+struct hse_mclass_info {
+    uint64_t mi_allocated_bytes; /**< Allocated storage space for a media class. */
+    uint64_t mi_used_bytes;      /**< Used storage space for a media class. */
+    uint64_t mi_reserved[4];     /**< Reserved space for future expansion. */
```

Comment:
I presume the reservation will be doubled here.

Reply:
Yes. Just haven't done it since our discussion in the RFC. Do you think the reserved spaces should go after the the path?

Reply:
What you have currently makes sense to me.

Reply:
Agree. Before path.